packages/uniforms/__suites__/ListAddField.tsx

```diff
@@ -0,0 +1,78 @@
+import { screen } from '@testing-library/react';
+import userEvent, {
+  PointerEventsCheckLevel,
+} from '@testing-library/user-event';
+import React, { ComponentType } from 'react';
+import z from 'zod';
+
+import { renderWithZod } from './render-zod';
+
+export function testListAddField(ListAddField: ComponentType<any>) {
+  test('<ListAddField> - correctly reacts on click', async () => {
```

Comment:
```suggestion
  test('<ListAddField> - calls onChange when clicked', async () => {
```

Reply:
I think that we should not change this because now consistent names with `ListDelField`.

Reply:
We've decided to review all test names once the migration to RTL is done.